common-arcana.lic

```diff
@@ -640,4 +640,23 @@ module DRCA
 
     crafting_prepare_spell(training_spells[needs_training], settings)
   end
+
+  def cast_waggle(settings, waggle_sets_name = 'default')
```

Comment:
💡 I'm not a fan of the default value. I think `buff` should be responsible for figuring out how to interpret the user's args. Thoughts?

Reply:
Done and done